plugins/woocommerce/includes/class-wc-comments.php

```diff
@@ -43,8 +43,11 @@ public static function init() {
 		add_filter( 'comments_clauses', array( __CLASS__, 'exclude_webhook_comments' ), 10, 1 );
 		add_filter( 'comment_feed_where', array( __CLASS__, 'exclude_webhook_comments_from_feed_where' ) );
 
+		// Secure potential remaining Action Logs.
+		add_filter( 'comments_clauses', array( __CLASS__, 'exclude_action_log_comments' ), 10, 2 );
+
 		// Exclude product reviews.
-		add_filter( 'comments_clauses', array( ReviewsUtil::class, 'comments_clauses_without_product_reviews' ), 10, 1 );
+		add_filter( 'comments_clauses', array( ReviewsUtil::class, 'comments_clauses_without_product_reviews' ), 10, 2 );
```

Comment:
Don't have full context here, but it seems that `comment_feed_where` was not added for the two new handlers of `comments_clauses`. If it's intentional, can we leave some comments on why we skipped them?